src/test/resources/sql/insert_cases.sql

```diff
@@ -40,45 +40,6 @@ VALUES (1, 'TestAddressBookCase', 'PROBATE', 'CaseCreated', 'PUBLIC',
        '2016-06-24 20:44:52.824'
 );
 
-INSERT INTO case_data (id, case_type_id, jurisdiction, state, security_classification, data, data_classification, reference, created_date, last_modified, last_state_modified_date)
-VALUES (180, 'TestCaseDocumentField', 'PROBATE', 'CaseCreated', 'PUBLIC',
-        '{
-          "PersonFirstName": "Janet",
-          "PersonLastName": "Parker",
-          "PersonAddress": {
-            "AddressLine1": "123",
-            "AddressLine2": "Fake Street",
-            "AddressLine3": "Hexton",
-            "Country": "England",
-            "Postcode": "HX08 UTG"
-          },
-          "D8Document": {
-            "document_url": "http://localhost:[port]/documents/05e7cd7e-7041-4d8a-826a-7bb49dfd83d0",
-            "document_binary_url": "http://localhost:[port]/documents/05e7cd7e-7041-4d8a-826a-7bb49dfd83d0/binary",
-            "document_filename": "Seagulls_Square.jpg"
-          }
-        }',
-       '{
-         "PersonFirstName": "PUBLIC",
-         "PersonLastName": "PUBLIC",
-         "PersonAddress": {
-           "classification" : "PUBLIC",
-           "value" : {
-             "AddressLine1": "PUBLIC",
-             "AddressLine2": "PUBLIC",
-             "AddressLine3": "PUBLIC",
-             "Country": "PUBLIC",
-             "Postcode": "PUBLIC"
-           }
-         },
-         "D8Document": "PUBLIC"
-       }',
-       '1504259907353520',
-       '2016-06-22 20:44:52.824',
-       '2016-06-24 20:44:52.824',
-       '2016-06-24 20:44:52.824'
-);
-
```

Comment:
not sure why this is showing as it is something i added into my branch and not 588.

anyway, it was just me making sure it wasn't the sql entry that was causing the issue.

Reply:
showing as outdated now, never mind this.